packages/uicore/src/components/DropDown/DropDown.tsx

```diff
@@ -175,6 +175,7 @@ export const DropDown: FC<DropDownProps> = props => {
     debounce(query => {
       if (Array.isArray(items)) {
         setDropDownItems(items.filter(item => item.label.toLowerCase().includes(query.toLowerCase())))
+        onQueryChange?.(query)
```

Comment:
Nit: `else if` can be removed so that `onQueryChange` can be written once (before line 182)

Reply:
i am not sure of the use case which else if carries across.
Should i remove it?